### PR TITLE
Documentation page

### PR DESCRIPTION
**Pull request by jtomeck**
@manthey - Here is the documentation page based on your document. I've optimized all the png's and also I have noticed there are a lot of broken links. I've indicated that with `-- BROKEN LINK --` next to each one.

### CODE REVIEW

**Comment by manthey** [comment]
I think a lot of those links will become valid as soon as https://github.com/DigitalSlideArchive/HistomicsTK/pull/842 is merged.

**Comment by jtomeck** [comment]
Okay I'll remove all the indicators for broken links that I added. Thanks!

**Comment by manthey** [comment]
@cooperlab Commented via email: 

Looks nice.​

I suggest linking the topic titles and images to the target notebooks too. The links in the text are small and easy to miss.

In TOC making the themes larger or bold to help them stand out.

File: docs/documentation/index.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
+---
+layout: page
+permalink: /documentation/
+toc: true
+
+# Banner Content
+# =================================
+title: DSA/HistomicsTK Documentation
```
Comment (manthey) [inline docs/documentation/index.md:8]: Because of the font we use, I think this would scan better with spaces around the `/` (i.e., `DSA / HistomicsTK`).

**Comment by manthey** [comment]
> Oh yeah, we also need to link to it somewhere.

Do you think that an appropriate place would be below the dual column area of HistomicsUI/HistomicsTK and above the Demos/Stores/Papers, with a largish text link that reads "Managing Annotation Projects, Processing Annotations and Mask, and Performing Image Analysis"?

**Comment by faiza-a** [comment]
Page looks awesome!

**Comment by kheffah** [comment]
@cooperlab @dgutman I believe we should merge this ASAP if no further edits are needed. But first, let's merge PR #112 as suggested by David M.

**Comment by kheffah** [comment]
@faiza-a Please don't merge this before merging PR #112 here first.

**Comment by kheffah** [comment]
@cooperlab @dgutman When you get time, please merge PR #112 first here, then merge this PR into master. Here's a local demo of what the website will look like after the merge: https://youtu.be/zuZ2eRtXXTM

**Pull request merged**